python/docs/source/conf.py

```diff
@@ -61,9 +61,9 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'sbp'
-copyright = u'2015, Swift Navigation'
-author = u'Swift Navigation'
+project = 'sbp'
+copyright = '2015, Swift Navigation'
```

Comment:
Update year? `2015, 2019`

Reply:
This PR doesn't make any significant changes to the documentation, so I don't think I should update the year here. If the swift guys wanted to update the year to either "2015 ,2019" or "2015-2019", it would be better done in conjunction with generating the next release.